src/components/Identicon.vue

```diff
@@ -36,11 +24,31 @@ export default class Identicon extends Vue {
     private dataUrl: string = this.placeholderDataUrl;
 
     @Watch('address', { immediate: true })
-    private computeDataUrl(address: string, oldAddress?: string) {
+    private async computeDataUrl(address: string, oldAddress?: string) {
         if (this.address && Identicon.isUserFriendlyAddress(this.address)) {
-            Iqons.toDataUrl(Identicon.formatAddress(this.address)).then((dataUrl: string) => {
-                this.dataUrl = dataUrl;
-            });
+            // Set svgPath
+            // @ts-ignore
+            if (self.NIMIQ_IQONS_SVG_PATH) {
+                console.warn('Setting NIMIQ_IQONS_SVG_PATH is deprecated and support will be removed in the future. '
+                    + 'Use setAssetPublicPath instead.');
+                // @ts-ignore
+                Iqons.svgPath = self.NIMIQ_IQONS_SVG_PATH;
+            } else {
+                // Use file-loader to copy the svg asset to dist and set the path to where the file is located. Instead
+                // of a normal import use a dynamic import at usage time to give apps the opportunity to adapt the base
+                // path via setAssetPublicPath before the path is being determined. Using webpackMode: 'eager' to avoid
+                // creating an additional chunk and to let the import resolve immediately.
+                let { default: svgPath } = (await import(
+                    /* webpackMode: 'eager' */
+                    '!!file-loader?name=[name].[hash:8].[ext]!@nimiq/iqons/dist/iqons.min.svg'));
+                if (typeof self.NIMIQ_VUE_COMPONENTS_IMAGE_ASSET_PATH === 'string') {
+                    // @ts-ignore
+                    svgPath = svgPath.replace(__webpack_public_path__, self.NIMIQ_VUE_COMPONENTS_IMAGE_ASSET_PATH);
```

Comment:
Which TS error is ignored here?

Reply:
`__webpack_public_path__` is a global variable unknown to ts.